LondonBritishLibrary/orient/BLorient13263.xml

```diff
@@ -0,0 +1,150 @@
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">60</measure>
+                                    <measure unit="leaf" type="blank" corresp="#1r #2 #59v #60">4</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>100</height>
+                                        <width>60</width>
+                                    </dimensions>
+                                </extent></supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="13">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Very careful handwriting.</desc>
+                                <date notBefore="1700" notAfter="1799" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="OwnershipNote"/>
+                                    <q xml:lang="gez"><persName role="owner" ref="PRS14338SahlaMaryam">ሠህለ፡ ማርያም፡</persName></q>
+                                </item>
+                                <item xml:id="e2"> 
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="en">Presented by the <placeName ref="INS0775LWC">Wellcome Institute</placeName> 
+                                        in <date when="1970">1970</date>.</q>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1"/>
+                                <decoNote type="bindingMaterial" xml:id="b2"><material key="wood"/>Remains of a binding of wooden boards.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1700" notAfter="1799">18th century</origDate>
+                        </origin>
+                        <provenance>According to 
+                            <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">11</citedRange></bibl> the owner
+                            <persName ref="PRS14338SahlaMaryam">Śahla Māryām</persName> in inscribed and the book was presented by the 
```

Comment:
"is inscribed"? Maybe Strelcyn's "inscribed" is not an ideal term. I would possibly also split the provenance information into two sentences, because the original owner and the donation to the BL have no causal relationship.